flocker/node/_diagnostics.py

```diff
@@ -0,0 +1,313 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+A script to export Flocker log files and system information.
+"""
+
+from gzip import open as gzip_open
+import os
+from platform import dist as platform_dist
+import re
+from shutil import copyfileobj, make_archive, rmtree
+from socket import gethostname
+from subprocess import check_call, check_output
+from uuid import uuid1
+
+from flocker import __version__
+
+
+def gzip_file(source_path, archive_path):
+    """
+    Create a gzip compressed archive of ``source_path`` at ``archive_path``.
+    An empty archive file will be created if the source file does not exist.
```

Comment:
It's bad I know. 

My reason is that there won't be any `flocker-*_startup` logs on Ubuntu unless there are (or have been) errors during service startup.
I wanted a consistent set of files in the diagnostic archive for testing purposes.